ddtrace/tracer.py

```diff
@@ -681,7 +681,7 @@ def _initialize_span_processors(self, appsec_enabled=asbool(get_env("appsec", "e
                 from .appsec.processor import AppSecSpanProcessor
 
                 appsec_span_processor = AppSecSpanProcessor()
-                self._span_processors.append(appsec_span_processor)
+                self._span_processors.insert(0, appsec_span_processor)
```

Comment:
I'd rather we initialize processors in order (something like how we do the trace processors)

This way it's more clear that `SpanAggregator` is the last in the list.

Reply:
for example:

```python
processors = []

if app_sec_enabled:
    processors.append(AppSecSpanProcessor())

processors.append(SpanAggregator())
```